src/execution/execute.js

```diff
@@ -916,6 +918,58 @@ function completeValue(
   );
 }
 
+/**
+ * Complete a async iterable value by completing each item in the list with
+ * the inner type
+ */
+
+function completeAsyncIterableValue(
+  exeContext: ExecutionContext,
+  returnType: GraphQLList<GraphQLOutputType>,
+  fieldNodes: $ReadOnlyArray<FieldNode>,
+  info: GraphQLResolveInfo,
+  path: Path,
+  result: AsyncIterable<mixed>,
+): Promise<$ReadOnlyArray<mixed>> {
+  // $FlowFixMe
+  const iteratorMethod = result[SYMBOL_ASYNC_ITERATOR];
+  const iterator = iteratorMethod.call(result);
+
+  const completedResults = [];
+  let index = 0;
+
+  const itemType = returnType.ofType;
+
+  const handleNext = () => {
+    const fieldPath = addPath(path, index);
```

Comment:
Can you please check if you need to add third argument.

Reply:
I'm not sure I understand what the third argument is for and when it should be passed. I updated it to pass `undefined` as that's what's done currently for list values: https://github.com/graphql/graphql-js/blob/6012d28b3b1fe8e002eefed9a3f013d4b352f08d/src/execution/execute.js#L944